neuraxle/checkpoints.py

```diff
@@ -88,6 +88,8 @@ def is_for_execution_mode(self, execution_mode: ExecutionMode) -> bool:
                 ExecutionMode.FIT_OR_FIT_TRANSFORM_OR_TRANSFORM
             ]
 
+        return execution_mode == ExecutionMode.FIT_OR_FIT_TRANSFORM_OR_TRANSFORM
```

Comment:
We'll need to refactor execution modes at some point.

Reply:
created issue #206